components/typeform/actions/common.mjs

```diff
@@ -1,4 +1,9 @@
+import typeform from "../typeform.app.mjs";
+
 export default {
+  props: {
+    typeform,
+  },
   methods: {
     addProperty({
```

Comment:
The methods (`reduceProperties` and `addProperty`) in this common file could also be moved to a `utils.mjs` or `common/utils.mjs` file since they're used only in two of the actions that include `actions/common.mjs`?